pkg/virt-api/rest/subresource.go

```diff
@@ -411,8 +416,13 @@ func (app *SubresourceAPIApp) RestartVMRequestHandler(request *restful.Request,
 		return
 	}
 
+	stopChangeRequestData := make(map[string](string))
```

Comment:
nit: unnecessary parenthesis
```suggestion
	stopChangeRequestData := make(map[string]string)
```